index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" >
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio </title>
+   <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css">
+    <link rel="stylesheet" href="style.css">
+  </head>
+      <body>
+      <div class="flex-container">
+        <div class="item1">
+          <header class="header">
+            <nav>
+            <p class="time"> 9:14 <i class='fas fa-wifi'></i></p>
+            <button class="button"><a href="#">Logo</a></button> 
+            <a href="#"><i class="fa fa-bars"></i></a>
+            </nav>
+         </header>
+        </div>
+        <div class="item2">
+          <section>
+            <h1 class="mariane">I'm Mariane<br>
+              Glad to see<br>
+             you!<br>
+           </h1>
+          </section>
+            <section>
+            <p class="text">
+              I’m a software developer! I can help you<br>
+               build a product , feature or website Look <br>
+               through some of my work and<br>
+                experience! If you like what you see and <br>
+                have a project you need coded, don’t<br>
+                 hestiate to contact me.
+            </p>
+          </section>
+          <section class="cap">
+            <button>LET’S CONNECT</button>
+          </section>
```

Comment:
- [ ] It seems like the changes you introduced altered the look of this section. Kindly note that this is infact supposed to be a header (X) eg h2, h3 etc. Kindly also ensure that the color and font sizes match the Figma design.

**This is how I am seeing it right now:**

![image](https://user-images.githubusercontent.com/86318284/198292797-5ad97770-e429-4acf-8c2c-e50736ccbbad.png)

**Desired design:**

![image](https://user-images.githubusercontent.com/86318284/198292863-071e446b-5cf9-4db9-a7e4-48a1bcd6d6e4.png)